### ISSUE TITLE
One to One relationship

### ISSUE THREAD

**Issue by janantos** [otykier/TabularEditor#221]
HI,
The is not possible to edit exiting or create new relationship as with cardinality one to one. From/To cardinality in relations properties are all the times static and grayed out.

In fact only Many to one realtionship can be created, and no posibility to edit cardinality settings on existing ones.

 
![image](https://user-images.githubusercontent.com/19323286/49084621-4690cb80-f250-11e8-9dab-868355a2c030.png)

**Comment by otykier** [comment]
That's a bug. Good catch.

As a workaround, save your model as a .bim file after creating the relationship (as many-to-one). Note the ID of the relationship. Then, close Tabular Editor and open the .bim file in a text editor. Search for the ID of the relationship, and manually add the **fromCardinality** and **crossFilteringBehavior** properties to that section, like so:

```
{
    "name": "f684d854-335f-4f75-b71a-6d07900183e9",
    "fromTable": "Product 1",
    "fromColumn": "ProductKey",
    "toTable": "Product",
    "toColumn": "ProductKey",		
    "fromCardinality": "One",
    "crossFilteringBehavior": "bothDirections"
}
```

Save the file and then load it back into Tabular Editor. The next time you need to create a one-to-one relationship, you can simply copy/paste (CTRL+C/CTRL+V) the first relationship you created this way, and reassign the FromColumn and ToColumn properties within Tabular Editor.

**Comment by janantos** [comment]
For time of being, I have solved this using PowerShell and AMO but thanks.

**Comment by otykier** [comment]
Fixed in [2.8](https://github.com/otykier/TabularEditor/releases/tag/2.8)
